examples/webgl_postprocessing_pixel.html

```diff
@@ -0,0 +1,244 @@
+            }
+
+            #info {
+                color: gray;
+                position: absolute;
+                bottom: 0px;
+                width: 100%;
+                padding: 15px;
+            }
+            
+            .dg.ac {
+                z-index: 1 !important;
+            }
+        </style>
+    </head>
+    <body>
+
+        <script src="../build/three.js"></script>
+
+        <script src="js/shaders/CopyShader.js"></script>
+        <script src="js/shaders/PixelShader.js"></script>
+        <script src="js/postprocessing/EffectComposer.js"></script>
+        <script src="js/postprocessing/RenderPass.js"></script>
+        <script src="js/postprocessing/ShaderPass.js"></script>
+        <script src="js/controls/TrackballControls.js"></script>
+        <script src='js/libs/dat.gui.min.js'></script>
+
+        <div id="container"></div>
+
+        <div id="info">
+            <a href="http://threejs.org" target="_blank" rel="noopener">three.js</a> - pixel shader example<br/>
+            shader by <a href="http://wongbryan.github.io">wongbryan</a>
+        </div>
+
+        <script>
+
+            var camera, scene, renderer, gui, composer;
+            var pixelPass, params;
+
+            var geomData = [], matData = [], group;
+
+            function updateGUI(){
+
+                pixelPass.uniforms.pixels.value = params.pixels;
+                pixelPass.uniforms.step.value = params.step;
+            }
+
+            function resize(){
+
+                camera.aspect = window.innerWidth / window.innerHeight;
+                camera.updateProjectionMatrix();
+                renderer.setSize( window.innerWidth, window.innerHeight );
+            
+            }
+
+            function init() {
+
+                var container = document.getElementById( 'container' );
+                renderer = new THREE.WebGLRenderer({antialias: true});
+                renderer.shadowMap.enabled = true;
+                renderer.shadowMap.type = THREE.PCSoftShadowMap;
+                renderer.setPixelRatio( window.devicePixelRatio );
+                renderer.setSize( window.innerWidth, window.innerHeight);
+                renderer.setClearColor(0xbfe7ff);
+                container.appendChild( renderer.domElement );
+                
+                camera = new THREE.PerspectiveCamera( 75, window.innerWidth / window.innerHeight, 1, 10000 );
+                camera.position.set(0, 0, 30);
+                controls = new THREE.TrackballControls(camera, renderer.domElement);
+                controls.rotateSpeed = 2.0;
+                controls.panSpeed = 0.8;
+                controls.zoomSpeed = 1.5;
+
+                scene = new THREE.Scene();
+
+                var hemisphereLight = new THREE.HemisphereLight(0xfceafc, 0x000000, .8)
+
+                var shadowLight = new THREE.DirectionalLight(0xffffff, .5);
+
+                shadowLight.position.set(150, 75, 150);
+
+                shadowLight.castShadow = true;
```

Comment:
There are no objects receiving shadows in this demo. Shadows can be removed, I expect.